salem/sio.py

```diff
@@ -1079,32 +1162,46 @@ def open_metum_dataset(file, pole_longitude=None, pole_latitude=None,
             ds[vn] = v.where(v <= 180, v - 360)
 
     # get pyproj string
-    if pole_longitude is None or pole_latitude is None:
+    if pole_longitude is None or pole_latitude is None or central_rotated_longitude is None:
         # search for specific attributes names
         n_lon = 'grid_north_pole_longitude'
         n_lat = 'grid_north_pole_latitude'
+        cen_lon = 'north_pole_grid_longitude'
         # first in dataset
         pole_longitude = ds.attrs.get(n_lon, None)
         pole_latitude = ds.attrs.get(n_lat, None)
+        central_rotated_longitude = ds.attrs.get(n_lat, None)
         # then as variable attribute
-        if pole_longitude is None or pole_latitude is None:
+        if pole_longitude is None or pole_latitude is None or central_rotated_longitude is None:
             for k, v in ds.variables.items():
                 if n_lon in v.attrs:
                     pole_longitude = v.attrs[n_lon]
                 if n_lat in v.attrs:
                     pole_latitude = v.attrs[n_lat]
+                if cen_lon in v.attrs:
+                    central_rotated_longitude = v.attrs[cen_lon]
+                else:
+                    central_rotated_longitude = 0 #defaults to zero
                 if pole_longitude is not None and pole_latitude is not None:
                     break
 
-    srs = ('+ellps=WGS84 +proj=ob_tran +o_proj=latlon '
-           '+to_meter=0.0174532925199433 '
-           '+o_lon_p={o_lon_p} +o_lat_p={o_lat_p} +lon_0={lon_0} +no_defs')
-    params = {
-        'o_lon_p': central_rotated_longitude,
-        'o_lat_p': pole_latitude,
-        'lon_0': 180 + pole_longitude,
-    }
-    srs = srs.format(**params)
+    if automatic_proj:
```

Comment:
I'd prefer not to rely on cartopy (it is an optional dependency of salem). What is the advantage of this?